.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Run Tests
+    env:
+      TESTING: true
+      URL: localhost:5000
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)